code/__DEFINES/subsystems.dm

```diff
@@ -59,6 +59,7 @@
 #define INIT_ORDER_SOUNDS 16
 #define INIT_ORDER_INSTRUMENTS 15
 #define INIT_ORDER_RESEARCH 14 // SoonTM
+#define INIT_ORDER_STATION 13.5 //This is high priority because it manipulates a lot of the subsystems that will initialize after it.
```

Comment:
This bothers me a bit that this isnt an integer. It shouldnt really break anything, but I think it would be preferred if this was a whole number

Reply:
adjusted